fastexcel-writer/src/main/java/org/dhatim/fastexcel/Workbook.java

```diff
@@ -191,7 +192,41 @@ private void writeWorkbookFile() throws IOException {
             for (Worksheet ws : worksheets) {
                 writeWorkbookSheet(w, ws);
             }
-            w.append("</sheets></workbook>");
+            w.append("</sheets>");
+            /** Defining repeating rows and columns for the print setup...
+             *  This is defined for each sheet separately 
+             * (if there are any repeating rows or cols in the sheet at all) **/
+            worksheets.forEach( ws -> {
+                String defineName = "";
+                if(ws.getRepeatingCols() != null) {
+                    String startCol = ws.getRepeatingCols().get(0);
+                    String endCol = ws.getRepeatingCols().get(1);
+                    defineName += ws.getName() + "!$" + startCol + ":$" + endCol;
+                }
+                if(ws.getRepeatingRows() != null) {
+                    if (ws.getRepeatingCols() != null) defineName += ",";
+                    int startRow = ws.getRepeatingRows().get(0);
+                    int endRow = ws.getRepeatingRows().get(1);
+                    defineName += ws.getName() + "!$" + startRow + ":$" + endRow;
+                }
+                if (!defineName.equals("")) {
+                    try {
+                        w.append("<definedNames>");
+                        w.append("<definedName function=\"false\" " + 
+                                 "hidden=\"false\" " +
+                                 "localSheetId=\"0\" " + 
+                                 "name=\"_xlnm.Print_Titles\" " + 
+                                 "vbProcedure=\"false\">");
+                        w.append(defineName);
+                        w.append("</definedName>");
+                        w.append("</definedNames>");
+                    } catch (IOException e) {
```

Comment:
This method declares `throws IOException`, so don't catch it here. Just remove the `try` -  `catch`

Reply:
That will not work, it still requires handling an exception inside the forEach streams (https://stackoverflow.com/questions/29838774/unhandled-ioexception-inside-java-list-stream-foreach-lambda-expression)
Maybe I should avoid forEach then

Reply:
Yes, use standard `for` loop instead of `forEach()`